feat: Allow detection of MITM HTTPS proxies like ZScaler (#30174)
electron/electron: docs/api/session.md

    * `hostname` String
    * `certificate` [Certificate](structures/certificate.md)
    * `validatedCertificate` [Certificate](structures/certificate.md)

>     * `isIssuedByKnownRoot` Boolean - `true` if Chromium recognises the root CA as a standard root. If it isn't then it's probably the case that this certificate was generated by a MITM proxy whose root has been installed locally. This is meaningless if the certificate was not trusted.

Reviewer: ```suggestion
    * `isIssuedByKnownRoot` Boolean - `true` if Chromium recognises the root CA as a standard root. If it isn't then this certificate's root has been installed locally, for example by a corporate proxy. This is meaningless if the certificate was not trusted.
```

Also, I'm not sure what "if the certificate was not trusted" here means. Something about `verificationResult`?

Author: @nornagon I just copied it from Chromium, but yes. How about this:

```suggestion
    * `isIssuedByKnownRoot` Boolean - `true` if Chromium recognises the root CA as a standard root. If it isn't then it's probably the case that this certificate was generated by a MITM proxy whose root has been installed locally (for example, by a corporate proxy). This should not be trusted if the `verificationResult` is not `'OK'`.
```

>     * `verificationResult` String - Verification result from Chromium.

Author: ```suggestion
    * `verificationResult` String - Verification result from Chromium. `OK` if the certificate is trusted.
```
